packages/shadergradient/src/UI/Controls/Tools/AxisButton.tsx

```diff
@@ -53,12 +47,36 @@ export const IconButton: React.FC<ControlTypeTitlePropsT> = ({
       }}
       onClick={onClick}
     >
-      <Feather
-        /* @ts-ignore */
+      {/* <Feather
         name={icon}
         size={20}
         color={active ? 'white' : 'rgb(255,67,10)'}
-      />
+      /> */}
+      <svg xmlns='http://www.w3.org/2000/svg' width='24' height='24'>
+        <path
+          d='M 0 0 L 0 8.5'
+          transform='translate(8 12) rotate(60 0.5 4.25)'
+          fill='transparent'
+          stroke-width='1.8'
```

Comment:
need to change SVG property for React. @stone-skipper 

Please check console errors like `Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?`